dialogue-serde/build.gradle

```diff
@@ -22,6 +25,7 @@ dependencies {
     implementation 'com.palantir.conjure.java:conjure-lib'
     implementation 'com.palantir.safe-logging:safe-logging'
     implementation 'jakarta.ws.rs:jakarta.ws.rs-api'
+    compileOnly 'org.immutables:value::annotations'
```

Comment:
I feel like metric-schema should add this somehow?

Reply:
metric-schema doesn't use this, I think it may come from tritium though

Reply:
oh interesting